lib/flop_phoenix.ex

```diff
@@ -99,7 +99,7 @@ defmodule Flop.Phoenix do
   the `:event` option.
 
       def handle_event("paginate-pets", %{"page" => page}, socket) do
-        flop = Flop.set_page(socket.assigns.meta.flop, page)
+        params = Flop.set_page(socket.assigns.meta.flop, page)
```

Comment:
Thanks. I believe this line was correct, though, and the variable name should be changed to `flop` in line 104.